internal/cli/terraform.go

```diff
@@ -268,3 +278,49 @@ func terraformProviderCredentialsAreAvailable() bool {
 
 	return (domain != "" && clientID != "" && clientSecret != "") || (domain != "" && apiToken != "")
 }
+
+func checkOutputDirectoryIsEmpty(cli *cli, outputDIR string) bool {
+	_, err := os.Stat(outputDIR)
+	if os.IsNotExist(err) {
+		return true
+	}
+
+	_, mainFileErr := os.Stat(path.Join(outputDIR, "main.tf"))
+	_, importFileErr := os.Stat(path.Join(outputDIR, "auth0_import.tf"))
+	_, generatedFileErr := os.Stat(path.Join(outputDIR, "generated.tf"))
+	if os.IsNotExist(mainFileErr) && os.IsNotExist(importFileErr) && os.IsNotExist(generatedFileErr) {
+		return true
+	}
+
+	cli.renderer.Warnf(
+		"Output directory %q is not empty. "+
+			"Proceeding will overwrite the main.tf, auth0_import.tf and generated.tf files.",
+		outputDIR,
+	)
+
+	if !cli.force && !cli.noInput {
```

Comment:
Should we use `canPrompt(cmd)`?

Reply:
[`1d19261` (#798)](https://github.com/auth0/auth0-cli/pull/798/commits/1d1926139a41599c810cc98508b83a069eda3aa8) Updated!